jira.js

```diff
@@ -123,7 +123,36 @@ router.post('/webhook', async (req, res) => {
 			link.finished = true;
 			link.save();
 		} else if (req.body.transition && req.body.transition.transitionName === 'Send to Ikari') {
-			clipRequest(['youtube', req.body.issue.fields.customfield_10200, req.issue.fields.customfield_10201, req.issue.fields.summary, 'mkv']);
+			clipRequest(['youtube', req.body.issue.fields.customfield_10200, req.issue.fields.customfield_10201, req.issue.fields.summary, 'mkv'])
+				.then((promise) => {
+					if (promise === true) {
+						axios.post(`${url}/issue/${link.jiraId}/transitions`, {
+							transition: 41,
```

Comment:
```suggestion
							transition: {
							     id: '41' ,
							 },
```
Following schema: [Documentation](https://docs.atlassian.com/software/jira/docs/api/REST/8.5.13/#api/2/issue-doTransition:~:text=%2C-,%22transition%22%3A%20%7B,%7D%2C,-%22) 
Can also use name, see schema